src/main/java/mate/academy/internetshop/model/User.java

```diff
@@ -0,0 +1,82 @@
+package mate.academy.internetshop.model;
+
+import java.util.List;
+import java.util.Objects;
+
+public class User {
+
+    private Long id;
+    private String name;
+    private Order order;
+    private Bucket bucket;
+    private List<Order> orderList;
```

Comment:
this fields are redundant here

Reply:
the buyer may have several orders, for example by bank transfer and cash, or the history of his orders, so I need a list of his orders in class User

Reply:
@MateAcademy we already have a relation between order and user by userId in the order, so this relation is redundant, same about bucket

Reply:
I fixed it too